docs/20---Concepts/concepts-26572ad.md

```diff
@@ -158,6 +158,16 @@ You can use the service with multitenant applications developed in the context o
 
 
 
+</dd>
+</dl>
+
+**Security**
+
+</b></dt>
+<dd>
+
+The service has to be bound to the calling application in the same Cloud Foundry Space. Otherwise or when calling public APIs, an invalid authorization header (JWT token) will be sent which can lead to technical problems (e.g. for CAP applications).
+
```

Comment:
Thank you for your contribution! We have updated the content slightly. Before we merge the pull request, could you please let us know if you agree with the proposed changes?

```suggestion
</dd>
</dl>

**Security**

</b></dt>
<dd>

You have to bind the Job Scheduling service instance to the application in the same Cloud Foundry space. When the service calls an endpoint, the request always contains the Authorization header (JWT token). To validate this token, make sure that your application is bound to the Job Scheduling service instance.

For more information, see [Binding Service Instances to Applications](https://help.sap.com/docs/btp/sap-business-technology-platform/binding-service-instances-to-applications?version=Cloud).

```

Reply:
Hi @danachkov 

This looks good and is written more general, thanks.

Best regards,
Ben